Tools/Scripts/filter-test-logs

```diff
@@ -94,18 +92,24 @@ def filter_webdriver_tests(log_file_name):
             if 'tests ran as expected' in line:
                 print_line = True
                 print(line, end='')
+    return print_line
 
 def main():
     args = parser()
+    log_file = os.path.abspath(f'{args.output}')
     if args.filter_type == 'jsc':
-        filter_jsc_tests(args.output)
+        printed_results = filter_jsc_tests(log_file)
     elif args.filter_type == 'layout':
-        filter_layout_tests(args.output)
+        printed_results = filter_layout_tests(log_file)
     elif args.filter_type == 'scan-build':
-        filter_scan_build(args.output)
+        printed_results = filter_scan_build(log_file)
     elif args.filter_type == 'webdriver':
-        filter_webdriver_tests(args.output)
-
+        printed_results = filter_webdriver_tests(args.output)
+    if not printed_results:
+        with open(log_file, 'r') as raw_output:
+            for l in iter(deque(raw_output, maxlen=500)):
```

Comment:
`iter` is redundant (the `for` statement calls it implicitly).

Reply:
Oops, thanks!